editor/src/clj/editor/gui.clj

```diff
@@ -1552,9 +1555,9 @@
                                             [:build-targets :dep-build-targets])))
             (dynamic error (g/fnk [_node-id texture]
                              (prop-resource-error _node-id :texture texture "Texture")))
-            (dynamic edit-type (g/constantly
+            (dynamic edit-type (g/fnk [texture]
                                  {:type resource/Resource
-                                  :ext ["atlas" "tilesource"]})))
+                                  :ext (workspace/get-atlas-resource-extensions (:workspace texture))})))
```

Comment:
Here's a case of using the new public api `workspace/get-atlas-resource-extensions`